AnnoyancesFilter/MobileApp/sections/mobile-app_specific.txt

```diff
@@ -9,6 +9,9 @@
 !
 ! SECTION: Mobile apps - Regular rules
 goobike.com###appbnr
+openrec.tv##section[class^="Component__AppSuggestionCardSection"]
+openrec.tv##div[class^="AppBannerDialog__Wrapper"]
+openrec.tv##div[class^="AppBanner__Wrapper-"]
```

Comment:
You can do one rule instead of two
```
openrec.tv##div[class^="AppBanner"]
```

Reply:
Note, that the rules for mobile sites we add under the directive.

Reply:
I see. thank you for fixed my rule.